css-two-column-layout/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="row-header">
+      <div class="column-full">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p>We've dedicated thousands of man hours to making healthcare institutions in the Helium network run 10x more
+          efficiently and deliver and unparalleled quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-half">
```

Comment:
This `column-one-half` class has more `column` elements in it. There will never be a `column` directly inside another `column`, this means you need another `row` inside this `column`